.storybook/main.js

```diff
@@ -8,7 +8,7 @@
 module.exports = {
   stories: ['../packages/**/*.stories.tsx'],
   addons: [
-    { name: '@storybook/addon-essentials', options: { actions: false, viewport: false } },
+    { name: '@storybook/addon-essentials', options: { viewport: false } },
```

Comment:
I enabled the actions add-on because it's a very useful plugin too see what a function returns